calibration/DOFSTK/CGINS-DOFSTK-02628__20210615.csv

```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2628,CC_frequency_offset,-8.052759e+002, source_file DOFST-K_SBE43F_SN_2628_Calibration_2021-06-15.zip > 2628.cal
+43-2628,CC_oxygen_signal_slope,2.9159e-004, use Soc(adj) on SOCAdjusted pdf
+43-2628,CC_residual_temperature_correction_factor_a,-4.456875e-003,
+43-2628,CC_residual_temperature_correction_factor_b,1.725470e-004,
+43-2628,CC_residual_temperature_correction_factor_c,-2.435236e-006,
+43-2628,CC_residual_temperature_correction_factor_e,3.600000e-002,
```

Comment:
Looks good.  
But the source file isn't just the .cal file, since SOC(adj) was used.  Shouldn't we list the SOC(adj) pdf as the source file since it is the latest cal file?

Reply:
Yup it's noted on the applicable cal coeff line CC_oxygen_signal_slope